spec/menu_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'menu'
+
+describe Menu do
+  it 'should be a hash' do
+    expect(subject.items.class).to eq Hash
+  end
+
+  it 'should contain symbols and integers as key value pairs' do
+    expect(subject.items.all? {|k,v| k.class == Symbol && v.class == Fixnum}).to be true
```

Comment:
Line is too long. [88/80]<br>Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.